db/migrate/20250120094641_add_email_to_ect_at_school_periods.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailToECTAtSchoolPeriods < ActiveRecord::Migration[8.0]
+  def change
+    add_column :ect_at_school_periods, :email, :string
```

Comment:
Could we add a unique constraint in some form on these? We might need to add an emails table though and make these references to that table as I'm not sure having them in place would allow it?  Any thoughts?

Reply:
Hey @tonyheadford  - as discussed have added constraints on the model and db

Reply:
As discussed we are leaving constraints alone for now